docs/pages/project/contributing/contributing-cypress.md

```diff
@@ -98,7 +98,7 @@ Else, just run:
 {% include code.html code="npm run cy:dev:open" %}
 
 {%include alert.html content="Note the difference between local dependencies for Integration vs End-to-End Tests" %}
-keep in mind that if running integration tests (tests in [./integration/integration/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/integration) folder) the server doesn't need to be running but for full blown end-to-end tests (tests in [./integration/e2e/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/e2e) folder) its recommended to run `make server` OR make sure a Meshery user build (see [Getting Started](/installation/quick-start.md)) is installed and running locally before choosing one of those tests.
+keep in mind that if running integration tests (tests in [./integration/integration/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/integration) folder) the server doesn't need to be running but for full blown end-to-end tests (tests in [./integration/e2e/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/e2e) folder) its recommended to run `make server` OR make sure a Meshery user build (see [Getting Started](../../installation/quick-start.md)) is installed and running locally before choosing one of those tests.
```

Comment:
@jerensl, given that we're no longer using Cypress, which directory should we be pointing to?

@rohitsrma, thanks! Instead of `../..`, will you use `{{site.baseurl}}`?

Reply:
From what I know all the e2e tests are inside this directory: https://github.com/meshery/meshery/tree/master/ui/tests/e2e

Reply:
Hii @leecalcote, I've made the changes. The links to [./integration/integration/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/integration) and [./integration/e2e/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/e2e) are also broken. Should we update these with:

https://github.com/meshery/meshery/tree/master/ui/cypress/e2e
https://github.com/meshery/meshery/tree/master/ui/cypress/e2e/integration

Or is there another file we should link to?

Reply:
Yes, @rohitsrma. Please use the location that @jerensl has provided.

Reply:
@leecalcote, what's the location of integration ([./integration/integration/](https://github.com/meshery/meshery/tree/master/ui/cypress/integration/integration)) tests?

Reply:
It doesn't matter, @rohitsrma. We're not using Cypress any longer.